platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminContext.java

```diff
@@ -273,7 +274,7 @@ private ImportMigrationConfigurationAdminEntry proxy(ImportMigrationEntry entry)
         throw new MigrationException(
             "Import error: failed to read configuration [%s]; %s.", path, e);
       } finally {
-        IOUtils.closeQuietly(is);
+        Closeables.closeQuietly(is);
```

Comment:
Because Apache decided to deprecate both closeQuietly() for input and output streams. I understand we should never use that for output streams but for input, I believe it is still warranted as the tr-with-finally will end up bubbling out any exceptions that comes out of closing which I don't care here. Guava is nice here and will actually also log the suppressed exceptions if you turn on its logs.